src/metatrain/utils/scaler.py

```diff
@@ -20,6 +21,10 @@ class Scaler(torch.nn.Module):
     (that removes the multi-dimensional "mean" across the composition space) and/or
     other additive models. See the `train_model` method for more details.
 
+    The scaling is performed per-atom, i.e., in cases where the targets are
+    per-structure, the standard deviation is calculated on the targets divided by
+    the number of atoms in each structure.
+
```

Comment:
Should this be an option? To enable other kinds of targets where the value actually is per-structure?